juanevillam/xtrovert
# Set Up Internationalization with next-intl

## Overview  
This PR sets up internationalization for the project using the `next-intl` package, enabling the application to support multiple languages (`en` and `es`).

---

## Key Changes  

### **Features**  
- Installed the `next-intl` package.  
- Configured the application to handle multiple locales using `next-intl`.  
- Updated necessary files and configurations based on:  
  - [Next.js Internationalization Documentation](https://next-intl-docs.vercel.app/docs/getting-started/app-router/with-i18n-routing)  
  - [Next.js Internationalization Example](https://next-intl-example-app-router.vercel.app/)  

---

## Reasons for Changes  
Adding internationalization broadens accessibility by allowing the application to support multiple languages, providing a better user experience for a diverse audience.

---

## How to Test  
1. Run the application locally.  
2. Verify that content displays correctly in both English (`en`) and Spanish (`es`).  
3. Test switching between locales to ensure proper functionality.  
4. Confirm that URLs reflect the selected locale appropriately.
